scripts/deploy_update.py

```diff
@@ -21,17 +19,14 @@
 HOST_REQUIREMENTS = ["jupyterlab", "nodejs", "pip", "'python>=3.7'"]
 
 # Run requirements for conda package
-RUN_REQUIREMENTS = ["cdms2", "ipywidgets", "jupyterhub", "'jupyterlab=1.2'", "nb_conda",
-                    "'libnetcdf=4.7.3'", "nb_conda_kernels", "'python>=3.7'", "tqdm", "vcs"]
-
-# base conda channels (always added)
-BASE_CHANNELS = "-c conda-forge"
+RUN_REQUIREMENTS = ["cdms2", "ipywidgets", "jupyterhub", "jupyterlab", "nb_conda",
+                    "nb_conda_kernels", "'python>=3.7'", "tqdm", "vcs"]
 
 # dev conda channels (for developer deployment)
-DEV_CHANNELS = "-c cdat/label/nightly"
+DEV_CHANNELS = "-c cdat/label/nightly -c conda-forge"
 
 # user conda channels (for stable deployment)
-USER_CHANNELS = "-c cdat/label/nightly"
+USER_CHANNELS = "-c cdat/label/nightly -c conda-forge"
```

Comment:
Should we be installing from the (potentially unstable) nightly channel for vcs?

Reply:
Yeah I know what you mean. I have a DEV_CHANNELS and USER_CHANNELS defined, the DEV is supposed to be using the latest nightly while USER would have a stable version. I had to set the current USER version to nightly because there were issues I was running into which are resolved in the nightly channel (Lina recommended I use nightly for now)l. I'll change this to the v82 channel when it's all ready.